ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -1063,6 +1063,47 @@ long getLogUsedCapacity() {
     return getAbsolutePositionInLogForOffset(getCurrentEndOffset(indexSegments), indexSegments);
   }
 
+  /**
+   * @return absolute end position (in bytes) of latest PUT record when this method is invoked.
+   * @throws StoreException
+   */
+  long getAbsoluteEndPositionOfLastPut() throws StoreException {
```

Comment:
What do you mean by "absolute" in this context?

Reply:
The class `Offset` contains offset within specific log segment. Here we need offset in entire log.